package.json

```diff
@@ -14,13 +14,13 @@
     "dist-ci": "npm-run-all dist:* dist:*:*",
     "dist:gulp": "gulp",
     "dist:docs": "gulp docs",
-    "dist:prettier": "prettier ./{dist,docs}/*.{scss,css,json,js} --write",
-    "dist:colors:prettier-sketchpalette": "prettier ./dist/*.sketchpalette --write --parser json",
+    "dist:format": "sewing-kit format",
+    "dist:colors:format-sketchpalette": "prettier ./dist/*.sketchpalette --write --parser json",
     "dist:colors:ase": "node ./scripts/ase-encode.js",
     "dist:colors:clr:exclude-from-ci": "ase2clr ./dist/colors.ase ./dist/colors.clr",
     "dist:colors:rename-palettes:exclude-from-ci": "renamer --find colors --replace Polaris ./dist/colors.{ase,clr,sketchpalette}",
     "lint": "npm-run-all lint:**",
-    "lint:js": "eslint . --format codeframe",
+    "lint:js": "sewing-kit lint",
     "lint:tokens": "yamllint ./tokens/*.yml",
```

Comment:
Prettier should now run over yml files. Does yamllint offer anything that prettier does not?

Reply:
Good point - I just removed yaml-lint!